GPU/GPUTracking/Merger/GPUTPCGMMerger.cxx

```diff
@@ -1978,30 +1996,16 @@ GPUd() void GPUTPCGMMerger::Finalize2(int nBlocks, int nThreads, int iBlock, int
   }
 }
 
-struct MergeLooperParam {
-  float absz;
-  float tgl;
-  float qpt;
-  float x;
-  float y;
-  unsigned int id;
-};
-
-GPUd() void GPUTPCGMMerger::MergeLoopers(int nBlocks, int nThreads, int iBlock, int iThread)
+GPUd() void GPUTPCGMMerger::MergeLoopersInit(int nBlocks, int nThreads, int iBlock, int iThread)
 {
-  if (iThread || iBlock) {
-    return;
-  }
-#ifndef GPUCA_GPUCODE
-  std::vector<MergeLooperParam> params;
   const float lowPtThresh = Param().rec.tpcRejectQPt * 1.1f; // Might need to merge tracks above the threshold with parts below the threshold
-  for (unsigned int i = 0; i < mMemory->nOutputTracks; i++) {
+  for (unsigned int i = get_global_id(0); i < mMemory->nOutputTracks; i += get_global_size(0)) {
     const auto& trk = mOutputTracks[i];
     const auto& p = trk.GetParam();
     const float qptabs = CAMath::Abs(p.GetQPt());
     if (trk.NClusters() && qptabs > 5.f && qptabs <= lowPtThresh) {
```

Comment:
You wrote before that considering tracks with pT<400 MeV, but the cuts here correspond to 50-200 MeV. 
Was this intentional?
Also, should scale with B.

Reply:
I had looser cuts for the MC study (also for x/y) and reduced them to reduce the combinatorics.
Actually, now that it is anyway so fast, I can try to loosen the initial cuts a bit to further boost the rejection.

<50 MeV doesn't play a role since we drop them anyway, so no need to merge them.
For scaling with B: agreed, but this actually applies to several places. Will fix them all in a follow-up PR.